contracts/deployment/SystemV2.sol

```diff
@@ -308,10 +313,12 @@ contract SystemV2 {
         core.grantLocker(address(gserl));
         core.grantLocker(address(pcvRouter));
         core.grantLocker(address(pcvGuardian));
+        core.grantLocker(address(daiNonCustodialPsm));
+        core.grantLocker(address(usdcNonCustodialPsm));
 
         core.grantMinter(address(grlm));
 
-        // Allocator config
+        /// Allocator config
```

Comment:
I'm so triggered by your use of triple / even though it's not natspec lol

Reply:
this is just the style we have throughout the codebase

Reply:
I'm fine to do // if not natspec, I just assumed if we used /// for natspec, we should use that same style for all comments. Whatever we land on as style we should enforce everywhere, which is why I made this change here.